{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2019-4310",
      "ASSIGNER": "cve@mitre.org"
    },
    "affects": {
      "vendor": {
        "vendor_data": [

        ]
      }
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [

          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/161036",
          "name": "ibm-guardium-cve20194310-info-disc (161036)",
          "refsource": "XF",
          "tags": [

          ]
        },
        {
          "url": "https://www.ibm.com/support/docview.wss?uid=ibm10960298",
          "name": "https://www.ibm.com/support/docview.wss?uid=ibm10960298",
          "refsource": "CONFIRM",
          "tags": [

          ]
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "IBM Security Guardium Big Data Intelligence 4.0 (SonarG) uses an inadequate account lockout setting that could allow a remote attacker to brute force account credentials. IBM X-Force ID: 161036."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [

    ]
  },
  "impact": {
  },
  "publishedDate": "2019-08-20T19:15Z",
  "lastModifiedDate": "2019-08-20T19:40Z"
}